src/main.rs

```diff
@@ -101,10 +118,23 @@ pub fn banana_hit_basket(
             let banana_radius = BANANA_HEIGHT / 2.0;
 
             if distance < basket_radius + banana_radius {
-                score.value += 1;
+                score.value += points.value;
                 commands.entity(banana_entity).despawn();
             }
         }
+        for (banana_bunch_entity, banana_bunch_transform, points) in banana_bunch_query.iter_mut() {
+            let distance = banana_bunch_transform
+                .translation
+                .distance(basket_transform.translation);
+
+            let basket_radius = BASKET_WIDTH / 2.0;
+            let banana_bunch_radius = BANANA_HEIGHT / 2.0;
+
+            if distance < basket_radius + banana_bunch_radius {
+                score.value += points.value;
+                commands.entity(banana_bunch_entity).despawn();
+            }
+        }
```

Comment:
Similarly to the above comment, I'm wondering if there could be a way that we combine this loop with the one above to just have a single for-loop that handles the logic for adding points for both single bananas and banana bunches. What do you think?